app/controllers/application_controller.rb

```diff
@@ -71,7 +71,7 @@ def enforce_user_permissions
     redirect_to administrator_dashboard_path if current_user&.administrator?
   end
 
-  def ensure_user_is_reviewer
```

Comment:
Names should tell you what they do. In this case the name says one thing but the code says another. I need the name "ensure_user_is_reviewer" so I'm changing this name. I'm *very* open for a better name change!